angular/src/lib/time-picker/time-picker-dropdown.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, HostBinding } from '@angular/core';
+
+@Component({
+  selector: 'cui-time-picker-dropdown',
+  template: `
+    <div class='cui-timepicker__dropdown'>
+      <div class="inline-flex">
+        <ng-content></ng-content>
+      </div>
+    </div>
+  `,
+  styles: []
+})
```

Comment:
we can remove wrapper div and bind to host the css class

Reply:
https://momentum.design/components/time-picker/code#30-minute-steps
Thanks for the reviewing. I follow the dom here. The host div has another class